webapp/pages/registration.vue

```diff
@@ -39,18 +40,25 @@ export default {
     registrationType() {
       if (!this.method) {
         return (
-          (this.publicRegistration && 'public-registration') ||
-          (this.inviteRegistration && 'invite-code') ||
-          'no-public-registration'
+          (this.publicRegistration && { method: 'public-registration', activePage: null }) ||
+          (this.inviteRegistration && { method: 'invite-code', activePage: null }) || {
+            method: 'no-public-registration',
+            activePage: null,
+          }
         )
       } else {
         if (
           this.method === 'invite-mail' ||
           (this.method === 'invite-code' && this.inviteRegistration)
         ) {
-          return this.method
+          if (this.method === 'invite-code' && this.inviteCode && this.nonce && this.email)
```

Comment:
The problem here is that `this.inviteCode`, `this.nonce`, and `this.email` just do not exist, because they are only local variables in `data`.

We just have overlooked this. 😅

Reply:
I’ve done an additional commit …

Reply:
But how did it work then when we tried it? @Tirokk